Source/WebCore/crypto/SubtleCrypto.cpp

```diff
@@ -233,6 +241,11 @@ static ExceptionOr<std::unique_ptr<CryptoAlgorithmParameters>> normalizeCryptoAl
             result = makeUnique<CryptoAlgorithmEcKeyParams>(params);
             break;
         }
+        case CryptoAlgorithmIdentifier::Ed25519: {
+            auto result = makeUnique<CryptoAlgorithmParameters>();
```

Comment:
Shouldn't we use params somehow?
If we should do makeUnique<CryptoAlgorithmParameters>(params), why are the test passing?

Reply:
I do not think this is correct to do so, we are probably losing CryptoAlgorithmParameters::name and CryptoAlgorithmParameters::identifier. This seems error prone especially since other code path are doing `makeUnique<CryptoAlgorithmParameters>(params);` like for CryptoAlgorithmIdentifier::RSAES_PKCS1_v1_5 case.